### ISSUE TITLE
history not persistent

### ISSUE THREAD

**Issue by gdavidss** [vipzhicheng/logseq-plugin-copy-ref-to-journal#14]
History does not persist if Logseq is closed and reopened.

**Comment by vipzhicheng** [comment]
Yes, for now, it's an easy solution, I want to get some feedback about if it is necessary to persist.

**Comment by gmalone** [comment]
I agree with the importance of a persistent history for Move Block.  It's a needless annoyance to have to re-enter information that is easily stored (or, at least, remembered for the session).  When doing "clean up" in Logseq (i.e. putting blocks where they should be), Move Block is the go-to tool.  Streamlining the process to make it faster would be very nice. 

Move Block is an essential plugin.  Kindly consider making it slightly better, and far more usable, with a persistent history.

I've also noticed since Logseq 0.8.18 that the moved block does not get visually removed (when choosing Stay on current page).  Requires a refresh now.  Didn't use to be that way.

Love Move Block.  I'm pretty sure I've donated for this plugin, but will do again now.  ☕  Thanks.
